CodeOfConduct.md

```diff
@@ -0,0 +1,101 @@
+  * Exercise consideration and respect in your speech and actions.
+  * Attempt collaboration & de-escalation before conflict.
+  * Refrain from demeaning, discriminatory, or harassing behavior and speech.
+  * Be mindful of your fellow participants. Alert community leaders if you notice a dangerous situation, someone in distress, or violations of this Code of Conduct, even if they seem inconsequential.
+  
+### Unacceptable Behavior
+
+To be specific, the following behaviors are unacceptable within our community:
+
+  * Anything in violation of our [Terms of Service](https://nelson.social/terms) including posting anything illegal in Canada.
+  * Violence, threats of violence or violent language directed against another person.
+  * Using slurs or racist, sexist, homophobic, transphobic, ableist or otherwise discriminatory or hateful jokes or language; promoting white supremacy, anti-Semitism, transphobia, castism or other hateful ideologies.
+  * Posting or displaying sexually explicit or violent material.
+  * Posting or threatening to post other people's personally identifying information ("doxing").
+  * Personal insults, particularly those related to gender, sexual orientation, race, religion, or disability.
+  * Unwelcome sexual attention. This includes, sexualized comments or jokes; unwelcomed sexual advances.
+  * Deliberate intimidation, stalking or following (online or in person).
+  * Debating moderation actions in public. If you disagree with how a moderator enforced this Code of Conduct you may direct-message the moderation team.
+  * Tone policing. Messages you see on nelson.social may violate the standards of civility we set on nelson.social, but you should not criticize users for their tone. If the user is a member of the nelson.social community, you should report them; if not, you should feel free to block them. In neither case should you criticize their tone.
+  * Advocating for, or encouraging, any of the above behavior.
+
+### Addressing Grievances
+
+If you feel you have been falsely or unfairly accused of violating this Code of Conduct, you should notify admin@nelson.social with a concise description of your grievance. Your grievance will be handled in due course by the moderation team.
+
+### Scope
+
+We expect all community participants to abide by this Code of Conduct in all community venues--online and in-person--as well as in all one-on-one communications pertaining to community business.
+
+This code of conduct and its related procedures also applies to unacceptable behavior occurring outside the scope of community activities when such behavior has the potential to adversely affect the safety and well-being of community members.
+
+### Contact
+
+Email [admin@nelson.social](mailto:admin@nelson.social) or contact one of the moderators:
+
+* [@blaine](https://nelson.social/web/@blaine)
+* [@matt](https://nelson.social/web/@matt)
+* [@till](https://nelson.social/web/@till)
+
```

Comment:
I'd like to add a note that we're aware that we're three privileged white dudes (though only one born-canadian! Yay im/migration!) and while we are happy to continue to pay for the service and volunteer our labour, we would like to divest ourselves of power and would gladly welcome volunteers (and eventually paid staff, if appropriate) to help or lead decision making. (Or something roughly to that effect?)

Reply:
Nice idea! Done in 1bbf80906e2ad2e2bc12d7b84a6e71e87bdb574c

I wonder if we could also add something like that to the `About nelson.social` section